iroha/src/torii.rs

```diff
@@ -32,7 +33,7 @@ pub struct Torii {
 impl Torii {
     /// Default `Torii` constructor.
     pub fn new(
-        url: &str,
+        (url, connect_url): (&str, &str),
```

Comment:
Why use tuple here?

Reply:
`clippy` complains about too many arguments for Torii. We will need decompose it soon.

Reply:
I think it's better to use `#[allow(..)]` then.